pkg/util/kubernetes/hpa/hpa_test.go

```diff
@@ -32,121 +32,264 @@ func (d *fakeDatadogClient) QueryMetrics(from, to int64, query string) ([]datado
 	return nil, nil
 }
 
-func newFakeConfigMapStore(t *testing.T, ns, name string, metrics []custommetrics.ExternalMetricValue) (custommetrics.Store, kubernetes.Interface) {
-	client := fake.NewSimpleClientset()
-	store, err := custommetrics.NewConfigMapStore(client, ns, name)
-	require.NoError(t, err)
-	err = store.SetExternalMetricValues(metrics)
-	require.NoError(t, err)
-	return store, client
-}
-
-func TestHPAWatcherGC(t *testing.T) {
-	testCases := []struct {
-		caseName string
+func TestHPAProcessor_UpdateExternalMetrics(t *testing.T) {
```

Comment:
This should be renamed to `TestProcessor_UpdateExternalMetrics` to match the type name `Processor`.